src/test/java/bio/terra/service/snapshot/SnapshotMinimalConnectedTest.java

```diff
@@ -138,6 +139,12 @@ public void testMinimal() throws Exception {
     assertThat("to table is right", relationshipModel.getTo().getTable(), equalTo("sample"));
     assertThat(
         "to column is right", relationshipModel.getTo().getColumn(), equalTo("participant_id"));
+
+    List<Map<String, Object>> result =
+        SnapshotConnectedTestUtils.getSnapshotTable(
+            summaryModel.getName(), "participant", bigQuerySnapshotProject);
+
+    assertThat("participant has the correct age", result.get(0).get("age"), equalTo("23"));
```

Comment:
It looks like this is testing code that's only used in test, is that correct?

Reply:
Yes, these are part of exercising the BQ functionality.